swap_meet/clothing.py

```diff
@@ -1,2 +1,14 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "",condition = 0):
+        super().__init__(category = "Clothing", condition = condition)
```

Comment:
Nice use of super! Here, it would be a good idea to not have `category` be a parameter in the `Clothing` constructor. We know that we always want `category` to be `"Clothing"`, so we shouldn't give the caller the illusion that they can change it.

Also a small style note: when using default/named parameters, do not put spaces before or after the equals sign. It should instead look like this:
`condition=0`.